packages/ckeditor5-list/docs/features/lists.md

```diff
@@ -34,6 +34,12 @@ Use the editor below to see the list style plugin in action.
 
 {@snippet features/lists-style}
 
+## Related features
+
+These features also provide similar functionality:
+* {@link features/todo-lists To-do lists} &ndash; This feature lets you create a list of interactive checkboxes with labels.
```

Comment:
Use the same form. Maybe just "Create a list of..." and then "Set indentation" or whatever?